packages/wonder-blocks-core/util/get-element-intersection.test.js

```diff
@@ -79,7 +79,13 @@ describe("getElementIntersection", () => {
             // First, let's fake out some scroll parent.
             // Flow doesn't like jest mocks $FlowFixMe
             enumerateScrollAncestors.mockImplementation(() => [
-                {getBoundingClientRect: jest.fn(() => rect(0, 100, 100, 0))},
+                {
+                    getBoundingClientRect: jest.fn(() => rect(0, 100, 100, 0)),
+                    currentStyle: {
+                        overflowX: "auto",
+                        overflowY: "auto",
+                    },
```

Comment:
I see why you used `currentStyle`. 🙂

Reply:
And yes, it also makes testing slightly easier (wouldn't be hard to have a mock for `getComputedStyle` though...and perhaps we should have a path that tests that.

Reply:
Nah, it's cool... you just be mocking it to run the same values.